README.md

```diff
@@ -1,2 +1,224 @@
+> - Kobiton's samples for automation test scripts in other languages: https://github.com/kobiton/samples
+
+**Automation test script execution**
+
+To execute the automation test script, just add a `run` section with command `npm run <SCRIPT_FILENAME>`. By default, it has already been added to the provided CircleCI configuration file.
+
+Example of executing the provided automation script:
+
+```yaml
+- run:
+    name: app-auto-testing
+    command: npm run automation-test-script
+    working_directory: samples/automation-test
+```
+
+### 5. Configuring HockeyApp WebHook listener
+Kobiton has provided script for setting up a small, basic server for receiving WebHook notifications from HockeyApp and triggering CircleCI to execute automation test of that application on Kobiton. The script will load settings from a configuration file and start a WebHook listener. This script must be executed on the server mentioned in **Prerequisites**.
+
+In order to use the provided script, please make some modifications to the configuration file:
+
+1. Open `/samples/webhook-server/config.yml`
+2. Replace those values with your CircleCI corresponding repository information:
+- `<PORT>`: Port for receiving WebHook requests (default: 3000).
+- `<YOUR_CIRCLECI_TOKEN>`: Your CircleCI personal token or project token of the corresponding repository.
+- `<VCS_TYPE>`: Your version control system (VCS) type (default: GitHub).
+- `<YOUR_PROJET_ORGANIZATION>`: Your repository organization.
+- `<YOUR_PROJECT_REPOSITORY>`: Your repository name.
+- `<PROJECT_BRANCH>`: Branch of the repository that contain the automation test script you want to execute (default: master).
+3. Execute the provided script using the below command:
+```bash
+ruby webhook_server.rb
+```
+
+4. In your application configuration in HockeyApp, change WebHook server address to your corresponding server address, port.
+
+5. Send a ping request from HockeyApp. If your server console output contains `Ping request received`, then you have successfully configured your server to listen for WebHook from HockeyApp.
+
+## C. Execution
+### 1. Executing the automation test
+
+Simply push your modified test script(s) and CircleCI configuration file `/.circleci/config.yml` to your GitHub repository. CircleCI will automatically configure the execution environment, install missing dependencies and execute the test script in `/samples` based on the provided configuration file `/.circleci/config.yml`.
+
+After you have pushed, the build process on CircleCI will end with error. That is normal, just simply ignore it.
+
+To execute the provided automation test script(s), simply upload your application package to HockeyApp. HockeyApp will process the uploaded file, send notification to the WebHook server, the server will then execute the automation test script on CircleCI with the URL received from HockeyApp WebHook notification.
```

Comment:
Please note that users can upload new app or new version of the app, please explain it more clearly.